### PR TITLE
chall01

### PR DESCRIPTION
**Pull request by J0NY97**
checks the requirements of the string passed in to the script.
prints out the corresponding morse code to the chars in the string.
if there is an error show usage.

### CODE REVIEW

File: chall01/jsalmi.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        's' : "...",
+        't' : "-",
+        'u' : "..-",
+        'v' : "...-",
+        'w' : ".--",
+        'x' : "-..-",
+        'y' : "-.--",
+        'z' : "--..",
+        ' ' : " "}
+if len(sys.argv) == 2 and len(sys.argv[1]) > 0 and all(i.lower() in alpha for i in sys.argv[1]):
```
Comment (jgengo) [inline chall01/jsalmi.py:32]: I don't understand your last check condition, why?

File: chall01/jsalmi.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    for letter in sys.argv[1]:
+        print(alpha[letter.lower()], end = '')
+    print('')
```
Comment (jgengo) [inline chall01/jsalmi.py:35]: you should avoid printing one by one and append it to a string then print the string.

File: chall01/jsalmi.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+else:
+    print ("usage:", sys.argv[0], "<a-zA-Z string>")
```
Comment (jgengo) [inline chall01/jsalmi.py:37]: good!

File: chall01/jsalmi.py (hunk 1 above)
Comment (J0NY97) [reply chall01/jsalmi.py:32]: It is to check that all of the characters in the input string is  one of the characters in the "alpha" dictionary.

File: chall01/jsalmi.py (hunk 2 above)
Comment (J0NY97) [reply chall01/jsalmi.py:35]: ok =)

File: chall01/jsalmi.py (hunk 3 above)
Comment (J0NY97) [reply chall01/jsalmi.py:37]: Thanks :)

**Pull request merged**